source/js/main.js

```diff
@@ -21,8 +22,14 @@ if (typeof chrome !== 'undefined') { // eslint-disable-line no-undef
       adReplacer(easylistSelectors, img)
     })
     $(window).scroll(() => {
-      adReplacer(easylistSelectors, img)
+      didScroll = true
     })
+    setInterval(() => {
+      if (didScroll) {
+        adReplacer(easylistSelectors, img)
+        didScroll = false
+      }
+    }, 10000)
```

Comment:
@autn is this every ten seconds? Maybe let's set to every half a second or so instead? What do you think?

Reply:
@YPCrumble yes, I think half a second is too fast. When we scroll, the content is almost loaded before, if we call the event too many times, the site will be "heavy". What do you think about 5 or 3 seconds?